include/odp/api/spec/timer.h

```diff
@@ -42,7 +42,9 @@ typedef enum {
 	/** Use CPU clock as clock source for timers */
 	ODP_CLOCK_CPU,
 	/** Use external clock as clock source for timers */
-	ODP_CLOCK_EXT
+	ODP_CLOCK_EXT,
+	/* Use external clock with constant frequency as clock source for timers */
+	ODP_CLOCK_EXT_CONST_HZ
```

Comment:
This feels unnecessary enumeration. The new external clock frequency parameter under can indicate if application is able to pass the clock frequency. This enumeration is for selecting between different clock sources (from multiple internal/external options).

Reply:
The idea was that EXT clock source might not have a fixed frequency, In such cases application should use ODP_CLOCK_EXT.
When EXT clock source is know to be of constant frequency ODP_CLOCK_EXT_CONST_HZ can be used as a hint to the underlying timer HW accelerator.